### PR TITLE
Fix http connection for healthchecker

### PR DESCRIPTION
**Pull request by jmoreira-valory**
* Override http connection to customize http ports. This is to avoid ports clashing with middleware ports.
* Add missing handlers for healthchecker.

@Divya-Solulab  @0xArdi Consider if these changes should apply to `optimus/service.yaml` as well.

### CODE REVIEW

**Review by gauravlochab — APPROVED** [review]

File: packages/valory/services/optimus/service.yaml [hunk 1]
```diff
@@ -128,3 +132,9 @@ cert_requests:
   not_before: '2022-01-01'
   public_key: ${ACN_NODE_PUBLIC_KEY:str:02d3a830c9d6ea1ae91936951430dee11f4662f33118b02190693be835359a9d77}
   save_path: .certs/acn_cosmos_11000.txt
+---
+public_id: valory/http_server:0.22.0
+type: connection
+config:
+  host: ${HTTP_SERVER_HOST:str:0.0.0.0}
+  port: ${HTTP_SERVER_PORT:int:8716}
```
Comment (jmoreira-valory) [inline packages/valory/services/optimus/service.yaml:140]: @DIvyaNautiyal07 @0xArdi Let me know if this is OK for the "normal" Optimus.

File: packages/valory/services/optimus/service.yaml [hunk 2]
```diff
@@ -6,9 +6,13 @@ aea_version: '>=1.0.0, <2.0.0'
...
 fingerprint_ignore_patterns: []
-agent: valory/optimus:0.1.0:bafybeida2scmw3qune3n6ru7tuzquuc3mxs2cfivzcncrtlj4ziadv4sqy
+agent: valory/optimus:0.1.0:bafybeid6iedxqwyvofrmcmihcsc66vxrcpjtoty7inf4bqtvjwbmbeehwm
 number_of_agents: 1
-deployment: {}
+deployment:
+  agent:
+    ports:
+      0:
+        8716: 8716
```
Comment (jmoreira-valory) [inline packages/valory/services/optimus/service.yaml:15]: @DIvyaNautiyal07 @0xArdi Let me know if this is OK for the "normal" Optimus.

File: packages/valory/services/optimus_pearl/service.yaml [hunk 3]
```diff
@@ -6,9 +6,15 @@ aea_version: '>=1.0.0, <2.0.0'
...
+agent: valory/optimus:0.1.0:bafybeie5vm4ucwtouxocxtd2aj6kvnrqsfngg32rhewvnzkfxnshqalepy
 number_of_agents: 1
-deployment: {}
+deployment:
+  agent:
+    ports:
+      0:
+        8716: 8716
+    volumes:
+      ./data: /data
```
Comment (dvilelaf) [inline packages/valory/services/optimus_pearl/service.yaml:17]: Probably not needed but no harm in it

File: packages/valory/services/optimus/service.yaml (hunk 1 above)
Comment (Divya-Solulab) [reply packages/valory/services/optimus/service.yaml:142]: @jmoreira-valory seems fine to me

**Review by Divya-Solulab — APPROVED** [review]

**Comment by dagacha** [comment]
any blocker to merge this @Divya-Solulab @gauravlochab ?

**Pull request merged**